http_check/datadog_checks/http_check/http_check.py

```diff
@@ -132,12 +133,17 @@ def get_ca_certs_path():
     Get a path to the trusted certificates of the system
     """
     """
-    check is installed via pip to embedded/lib/site-packages/datadog_checks/http_check
+    check is installed via pip to:
+    Windows: embedded/lib/site-packages/datadog_checks/http_check
+    Linux: embedded/lib/python2.7/site-packages/datadog_checks/http_check
     certificate is installed to   embedded/ssl/certs/cacert.pem
 
     walk up to embedded, and back down to ssl/certs to find the certificate file
     """
-    ca_certs = [os.path.normpath(os.path.join(os.path.dirname(__file__), '../../../..', 'ssl/certs', 'cacert.pem'))]
+    if Platform.is_windows():
+        ca_certs = [os.path.normpath(os.path.join(os.path.dirname(__file__), '../../../..', 'ssl/certs', 'cacert.pem'))]
+    else:
+        ca_certs = [os.path.normpath(os.path.join(os.path.dirname(__file__), '../../../../..', 'ssl/certs', 'cacert.pem'))]
```

Comment:
I'd remove the platform logic and just look for `embedded`:
```
embedded_root = os.path.dirname(os.path.abspath(__file__))
for _ in range(10):
    if os.path.basename(embedded_root) == 'embedded':
        break
    embedded_root = os.path.dirname(embedded_root)
else:
    raise OsError('Unable to locate `embedded` directory')

ca_certs = [os.path.join(embedded_root, 'ssl', 'certs', 'cacert.pem')]
```

Reply:
Nice call. Tweaked it ever so slightly, but updated with this logic.